### PR TITLE
LPS-108138 | LPS-108514 buildLang

### PR DESCRIPTION
**Pull request by gabrielpramos**

### CODE REVIEW

**Comment by gabrielpramos** [comment]
ci:test:sf

**Comment by gabrielpramos** [comment]
ci:test:relevant

**Comment by gabrielpramos** [comment]
ci:test:workflow

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/transition/bulk/select-transition-step/SelectTransitionStepBody.es.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import {capitalize} from '../../../../../../shared/util/util.es';
+import {Card} from './SelectTransitionStepCard.es';
+
+const Body = ({data, setRetry, tasks}) => {
+	const {workflowTaskTransitions = []} = data;
+
+	const taskSteps = useMemo(() => {
+		const versions = {};
+
+		tasks.forEach(task => {
```
Comment (albinojunior) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/transition/bulk/select-transition-step/SelectTransitionStepBody.es.js]: What do you think about to do deconstruct task?

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/transition/bulk/select-transition-step/SelectTransitionStepCard.es.js [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+						className="error-field"
+						data-testid="transitionSelect"
+						id={`transitionSelect${cardIndex}`}
+						onChange={handleChange}
+					>
+						<ClaySelect.Option label="" value="" />
+
+						{nextTransitions &&
+							nextTransitions.length > 0 &&
+							nextTransitions.map(nextTransition => (
```
Comment (albinojunior) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/transition/bulk/select-transition-step/SelectTransitionStepCard.es.js]: What do you think about this:
nextTransitions has a defaultValue as Array, you could to use only nextTransitions.map(...).

**Comment by albinojunior** [comment]
@gabrielpramos  Fix conflicting files and resend, please.

**Pull request closed without merging**